config: postpone creds applier when in RO till RW (#9119)
tarantool/tarantool: changelogs/unreleased/gh-8967-creds-postope-sync-when-RO-till-RW.md

## feature/config

> * If Tarantool is in Read Only mode, credentials are now synchronised
>   in the background when switched to Read Write mode, instead of being
>   skipped (gh-8967).

Reviewer: ```suggestion
* On read-only instances, Tarantool now synchronizes credentials
  in the background when switching to read-write mode instead of
  skipping the synchronization (gh-8967).
```

Author: Applied.
